tools/format_release_notes.sh

```diff
@@ -9,7 +9,7 @@ Usage: tools/format_release_notes.sh
 Note: this was written to work properly on mac. It should work on linux if -r is used instead of -E for sed.'
     exit 0
 fi
-evalml_version=$(sed -n -E "s/.*version=\'([0-9A-Za-z\.\_]+)\'.*/\1/p" setup.py)
+evalml_version=$(python -c "import evalml; print(evalml.__version__)")
```

Comment:
This assumes EvalML is installed where this is run. I think that's ok since this script is run by the devs but maybe it's better to use the `pep517.meta` trick Gaurav suggested here too.